README.md

```diff
@@ -2,71 +2,32 @@
 
 # Project Name
 
-> Description the project.
+> Hello microverse
```

Comment:
- [ ] Please add the Project Name as the title of the document and the description below it. You could change `# Project Name` to `# Hello Microverse` and add a description below it.